starboard/android/shared/audio_track_audio_sink_type.cc

```diff
@@ -273,11 +282,11 @@ void AudioTrackAudioSink::AudioThreadFunc() {
       }
     }
 
-    if (was_playing && !is_playing) {
-      was_playing = false;
+    if (audio_track_play_state == PLAYSTATE_PLAYING && !is_playing) {
       bridge_.Pause();
-    } else if (!was_playing && is_playing) {
-      was_playing = true;
+    } else if ((audio_track_play_state == PLAYSTATE_STOPPED ||
+                audio_track_play_state == PLAYSTATE_PAUSED) &&
```

Comment:
Would `audio_track_play_state != PLAYSTATE_PLAYING` be better here?

Reply:
Done.